pkg/api/install/install.go

```diff
@@ -34,35 +34,55 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
-// userResources is a group of resources mostly used by a kubectl user
-var userResources = []string{"rc", "svc", "pods", "pvc"}
-
 const importPrefix = "k8s.io/kubernetes/pkg/api"
 
 var accessor = meta.NewAccessor()
 
+// availableVersions lists all known external versions for this group from most preferred to least preferred
+var availableVersions = []unversioned.GroupVersion{v1.SchemeGroupVersion}
```

Comment:
The order used to be populated from the `registered` package, which is defined by the KUBE_API_VERSIONS environment variable, so user can change the order at runtime.

I guess you will have followup PR for register.go. Could you share your plan?